include/fcl/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+cmake_minimum_required(VERSION 3.6)
```

Comment:
It would be good to remove this line unless there is a compelling reason we need this. FCL is supposed to support Ubuntu 14.04 LTS, and the [CMake version on Ubuntu 14.04 LTS is 2.8.12](http://packages.ubuntu.com/trusty/cmake).